sbnanalysis/ana/SBNOsc/MakeHist/AcceptanceStack.C

```diff
@@ -0,0 +1,91 @@
+#include "TFile.h"
```

Comment:
It's helpful to include at the top of the file a multi-line `/** */` comment with:

* Name of the file
* Brief description
* Author name, email
* Longer description and how to use it
* Date of creation (and possibly a history of changes)

In this case, it might also be nice to include a README file in the `MakeHist` directory that provides some overall guidance to users about what's available.